testing/features/components/navigation.feature

```diff
@@ -13,10 +13,10 @@ Feature: Navigation component
     Scenario: All navigation links are shown
       Then the dropdown toggle is not present
 
+  @small_screen
   Rule: Default Navigation - small screens
```

Comment:
Oh nice, can we tag rules in Cucumber v7 then?

Reply:
Yep, It's all fixed now. Was good to get it here in the end.

Also v7 has a completely rewritten message buffer, which is now written using pure custom json, as opposed to using googles protobuf